application.yml

```diff
@@ -54,6 +54,19 @@ management:
       prometheus:
         enabled: true
 
+# Chaos Engineering
```

Comment:
The `run_all.sh` script starts the Spring Boot app with the profile `--spring.profiles.active=chaos-monkey`. Thus I propose to move this config section in a dedicated one like for `mysql`: 
```
spring:
  config:
    activate:
      on-profile: chaos-monkey
```

Reply:
Sure no problem